test/e2e/storage/persistent_volumes-local.go

```diff
@@ -1379,6 +1407,22 @@ func createProvisionerDaemonset(config *localTestConfig) {
 	framework.WaitForControlledPodsRunning(config.client, config.ns, daemonSetName, kind)
 }
 
+func findProvisionerDaemonsetPodName(config *localTestConfig) string {
+	podList, err := config.client.CoreV1().Pods(config.ns).List(metav1.ListOptions{})
+	if err != nil {
+		framework.Failf("could not get the pod list: %v", err)
+		return ""
+	}
+	pods := podList.Items
+	for _, pod := range pods {
+		if strings.HasPrefix(pod.Name, daemonSetName) && strings.HasPrefix(pod.Spec.NodeName, config.node0.Name) {
```

Comment:
Should the node name be an exact match instead of prefix?

Reply:
Changed to exact match.